ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -247,6 +244,20 @@ function WorkloadCvesOverviewPage() {
         }
     }
 
+    function onVulnerabilityStateChange(vulnerabilityState: VulnerabilityState) {
+        // Reset all filters, sorting, and pagination and apply to the current history entry
+        setActiveEntityTabKey('CVE');
```

Comment:
What are pro and con of comment that this initial change has push history action?

Reply:
Fair point - I suppose I used the comment here due to the newness of the pattern. I can see how as this becomes the standard the comment will be somewhat redundant.